app/graphql/mutations/delete_covid_library_mutation.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+module Mutations
+  # Mutation to delete a single ONT library
+  class DeleteCovidLibraryMutation < BaseMutation
+    argument :library_name, String, 'The name of the library to delete.', required: true
+
+    field :success, Boolean, 'Whether the library was successfully deleted.', null: false
+    field :errors, [String], 'An array of error messages thrown when deleting the library',
```

Comment:
I've been trying to put a full stop on the end of all the GraphQL documentation descriptions.